src/gui/tray/UserLine.qml

```diff
@@ -107,6 +107,14 @@ MenuItem {
                             font.pixelSize: 12
                             font.bold: true
                         }
+                        Label {
+                            id: accountStatus
+                            width: 128
+                            text: status

```

Comment:
I was wondering where status was coming from... but I see it's the same for all the User related stuff (like name just above). I smell qml file outreaching in the parent context or `model.` missing. Will need to be fixed but that's more a note for me to point out work will be needed, don't change it in this PR.